scripts/lib.sh

```diff
@@ -145,7 +145,10 @@ setup-git() {
         echo 'Please run `make setup-pyenv` to install the required Python 3 version.'
         exit 1
     )
-    pip install -r requirements-pre-commit.txt
```

Comment:
might be simpler to do `-r requirements-dev.txt` here -- would need fewer changes elsewhere

Reply:
Eh, I kind of wanna roll it all up into `make install-py-dev` once we make that fast.

Reply:
Actulaly nvm, need to do this to fix backend lint.